src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -460,6 +472,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return SyntaxFactory.ReferenceDirectiveTrivia(hashToken, referenceKeyword, file)
         End Function
 
+        Private Function ParseShebangDirective(hashToken As PunctuationSyntax) As ShebangDirectiveTriviaSyntax
+            Dim builder = New StringBuilder()
+            Debug.Assert(CurrentToken.Kind = SyntaxKind.ExclamationToken,
+                         NameOf(ParseShebangDirective) & " called with wrong token")
+            Dim exclamationToken = DirectCast(CurrentToken, PunctuationSyntax)
+            GetNextToken()
+
+            ' Eat tokens until the end of line.
+            While CurrentToken.Kind <> SyntaxKind.EndOfLineTrivia AndAlso
+                    CurrentToken.Kind <> SyntaxKind.EndOfFileToken AndAlso
+                    CurrentToken.Kind <> SyntaxKind.StatementTerminatorToken
+                builder.Append(CurrentToken.ToFullString())
+                GetNextToken()
+            End While
+
+
+            exclamationToken = exclamationToken.AddTrailingTrivia(SyntaxFactory.CommentTrivia(builder.ToString()))
+            Return SyntaxFactory.ShebangDirectiveTrivia(hashToken, exclamationToken)
+        End Function
+
+        Private Function ParseLoadDirective(hashToken As PunctuationSyntax) As LoadDirectiveTriviaSyntax
+            Debug.Assert(CurrentToken.Kind = SyntaxKind.IdentifierToken AndAlso DirectCast(CurrentToken, IdentifierTokenSyntax).PossibleKeywordKind = SyntaxKind.LoadKeyword,
+                         NameOf(ParseLoadDirective) & " called with wrong token")
+
+            Dim identifier = DirectCast(CurrentToken, IdentifierTokenSyntax)
+            GetNextToken()
+            Dim loadKeyword = _scanner.MakeKeyword(identifier)
+
+            If Not IsScript Then
+                loadKeyword = AddError(loadKeyword, ERRID.ERR_LoadDirectiveOnlyAllowedInScripts)
+            End If
+
+            Dim file As StringLiteralTokenSyntax = Nothing
+            VerifyExpectedToken(SyntaxKind.StringLiteralToken, file)
+
+            Return SyntaxFactory.LoadDirectiveTrivia(hashToken, loadKeyword, file)
```

Comment:
Yes, the rest of the line is treated as skipped text. I added relevant tests to check this behavior.
This is consistent with behavior of other directives. For instance you cannot type `#If DEBUG: Dim x = 12`.